airbyte-integrations/connectors/source-klaviyo/metadata.yaml

```diff
@@ -20,11 +20,21 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  suggestedStreams:
+    streams:
+      - events
+      - campaigns
+      - lists
+      - metrics
+      - flows
   releases:
     breakingChanges:
       1.0.0:
         message: In this release, for 'events' stream changed type of 'event_properties/items/quantity' field from integer to number. Users will need to refresh the source schema and reset events streams after upgrading.
         upgradeDeadline: "2023-11-30"
+      2.0.0:
+        message: In this release, for 'flows' stream changed cursor field from 'created' to 'updated'. Users will need to refresh the source schema and reset events streams after upgrading.
```

Comment:
In `abnormal_state.json` is see more streams being switched from `created` to `updated` cursor - is this really only affecting `flows` as is stated on this message or do we need to expand the description to include more streams?

Reply:
Nope, if they're _gaining_ incremental it's not breaking. I was confused because the test changes the cursor for more streams than just this one in the state json that's passed. But looking at the code it seems just this stream has changed so 👍🏼